cmd/savepcdfiles/main.go

```diff
@@ -69,10 +80,10 @@ func mainWithArgs(ctx context.Context, args []string, logger golog.Logger) error
 		Attributes: config.AttributeMap{"device_path": usbDevices[0].Path},
 	}
 
-	return savePCDFiles(ctx, int(argsParsed.Port), lidarDevice, logger)
+	return savePCDFiles(ctx, int(argsParsed.TimeDelta), int(argsParsed.Port), lidarDevice, logger)
```

Comment:
[q] Do you need to cast the `argsParsed.TimeDelta` to `int` here? I would assume it's already an integer based on it definition?

Reply:
resolved! I was copying what was done in Port, but I realized thats because it was cast to a special type